packages/components/src/base-control/index.js

```diff
@@ -8,6 +8,20 @@ import classnames from 'classnames';
  */
 import VisuallyHidden from '../visually-hidden';
 
+/**
+ * @typedef Props
```

Comment:
is this type "local"? I mean the name is very generic, is it fine?

Reply:
It is local to the module, although it is an exported type. It's pretty common practice to name types like this in my experience.